code/API_definitions/qod-api.yaml

```diff
@@ -40,8 +40,35 @@ paths:
         required: true
       callbacks:
         notifications:
-          "{$request.body#/notificationUrl}/notifications":
-            $ref: "#/paths/~1notifications"
+          "{$request.body#/webhook/notificationUrl}/notifications":
+            post:
+              tags:
+                - Session notifications callback
+              summary: "Session notifications callback"
+              description: |
+                Important: this endpoint is to be implemented by the API consumer.
+                The QoD server will call this endpoint whenever any network related event occurs.
+                Currently only QOS_STATUS_CHANGED event is defined.
+              operationId: postNotification
+              requestBody:
+                required: true
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/EventNotification"
+              responses:
```

Comment:
@sfnuser for which case do you think we need that? Usually you have it for operations where a specific server resource is referenced

Reply:
@akoshunyadi - The API server does not know the sanity/availability of the client service represented by the `notificationUrl`. Hence, I believe 404 should also be considered as an expected response.

Reply:
@sfnuser For that case '503 - Service unavailable' is defined

Reply:
@akoshunyadi - Sorry I was away. 503 is when the service is down. 404 is when there is a mistake in the service URL like a typo, the API gateway will return a 404 response.

Reply:
@sfnuser we define one endpoint with it's response codes, like we do it for our endpoints. So if another endpoint (a wrong one) is called, then other responses are possible. I'm not sure that we should discuss it here, since this PR is about to align with the guidelines and there is no 404 defined.